src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -376,8 +391,17 @@ public void onError(final String message)
             @Override
             public void execute()
             {
-               globalDisplay_.showErrorMessage("Error Saving File",
-                                               message);
+               // do not show the error if it is a transient autosave related issue - this can occur fairly frequently
+               // when attempting to save files that are being backed up by external software
+               if (message.contains("The process cannot access the file because it is being used by another process") && suppressFileLockError_)
```

Comment:
Checking this based on the message rather than an error property or code seems fragile. Would it be feasible for us to send a proper error object (e.g. with an error code) that we could check here or something similar?

Reply:
I decided against that because we are using the `ProgressIndicator` interface here which is tied to error messages and is a very fundamental interface in the code. I didn't think extending this interface was worth it to propagate this one error code here as this would affect a very large portion of the code base.